ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -460,6 +460,9 @@ keys:
             ipv4:
               type: dict
               keys:
+                border_router:
+                  type: bool
+                  description: Configure pim border router. EOS default is false
```

Comment:
Add a `.` at the end. And PIM in caps (everywhere)

Reply:
done